tests/cpp_unit_tests/test_y_bus.cpp

```diff
@@ -272,6 +274,181 @@ TEST_CASE("Test fill-in y bus") {
     CHECK(ybus.map_lu_y_bus == map_lu_y_bus);
 }
 
+TEST_CASE("Incremental update y-bus") {
+    /*
+    test Y bus struct
+    [
+            x, x, 0, 0
+            x, x, x, 0
+            0, x, x, x
+            0, 0, x, x
+    ]
+
+     [0]   = Node
+--0--> = Branch (from --id--> to)
+ -X-   = Open switch / not connected
+
+    Topology:
+
+  --- 4 ---               ----- 3 -----
+ |         |             |             |
+ |         v             v             |
+[0]       [1] --- 1 --> [2] --- 2 --> [3]
+ ^         |             |
+ |         |             5
+  --- 0 ---              |
+                         X
+    */
```

Comment:
as a good practice, it's better to use `//` even for multiline comments than `/* ... */`. The `/* ... */` are best used for inline comments and nowhere else. I can explain to you in detail but the reason comes down to commenting out blocks and then uncommenting them. most IDE's don't handle multi-line cases like

```cpp
/*
something
*/
```
correctly by default and you can accidentally do something like

```cpp
/*
// something
// */
```

and everything will break terribly. if you realize your error and try to uncomment using the same IDE command, it will actually uncomment like this in some cases:

```cpp
/*
// something
*/
```
or worse, depending on IDE.

Some users work around this by using `*` as a prefix, like

```cpp
/*
 * something
 */
```
and this will usually also work, but for some IDEs and formatters, this is also hard to deal with

multiline `//` always work exactly as you expect (because they use `\n` as a delimiter), and `/* ... */` used inline will also work correctly in all cases

Reply:
😉I blame formatter for this, I cursed like 2000 words after it destroyed my ASCII art and now I have accepted this intrusive behavior from the formatter as a new normal. Spoiler, the destroyed art came from the very first y-bus test case 🤐

Reply:
🗿